.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,65 @@
+name: Create Release PR
+on: 
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: "New sdk version"
+        required: true
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write
+        pull-requests: write
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with: 
+        ref: ${{ github.head_ref }}
+    - name: validate package dependencies
+      run: |
+        RESULT=$(swift package show-dependencies | grep unspecified)
```

Comment:
This verifies the package dependencies are all set to sem-ver. If they are not set to sem-ver, downstream users will be blocked from deploying releases to Apple.